src/Chapter2.hs

```diff
@@ -741,7 +769,11 @@ the list with only those lists that contain a passed element.
 
 🕯 HINT: Use the 'elem' function to check whether an element belongs to a list
 -}
-contains = error "contains: Not implemented!"
+contains :: Eq a => a -> [[a]] -> [[a]]
+contains _ [] = []
+contains x (y:ys)
+  | x `elem` y  = y : contains x ys
+  | otherwise   = contains x ys
```

Comment:
You can use `filter` function that already does exactly the logic you describe here 🙂 👍🏼